browser-test/src/trusted_intermediary.test.ts

```diff
@@ -59,9 +59,20 @@ test.describe('Trusted intermediaries', () => {
       lastName: 'last',
       dobDate: '1870-07-11',
     }
-    await tiDashboard.createClient(client)
+    await page.getByRole('button', {name: 'Add new client'}).click()
+    await waitForPageJsLoad(page)
+
+    await page.fill('#email-input', client.emailAddress)
+    await page.fill('#first-name-input', client.firstName)
+    await page.fill('#middle-name-input', client.middleName)
+    await page.fill('#last-name-input', client.lastName)
+    await page.fill('#date-of-birth-input', client.dobDate)
+
+    await page.getByRole('button', {name: 'Save'}).click()
+    await validateScreenshot(page, 'add-client-invalid-dob')
+    await tiDashboard.gotoTIDashboardPage(page)
+    await waitForPageJsLoad(page)
     await tiDashboard.expectDashboardNotContainClient(client)
-    await validateScreenshot(page, 'dashboard-add-client-invalid-dob')
   })
 
   test('expect Dashboard Contain New Client', async () => {
```

Comment:
suggestion (blocking):  I don't see the add client success alert in any of the snapshots.  Could you add it in this test or one of the others where we add a new client?